packages/terra-action-header/CHANGELOG.md

```diff
@@ -3,10 +3,12 @@ ChangeLog
 
 Unreleased
 ----------
+### Changed
+* Updated `--terra-action-header-icon-top` variable to only apply in Microsoft browsers.
```

Comment:
Let's update this to say something along the lines of "Removed `--terra-action-header-icon-top` variable, added `--terra-action-header-icon-ms-top` variable to only apply top styling in Microsoft browsers.

Reply:
Updated in d761f5e